src/app/backend/resource/node/list.go

```diff
@@ -100,11 +104,17 @@ func toNodeList(nodes []api.Node, dsQuery *dataselect.DataSelectQuery, heapsterC
 	return nodeList
 }
 
-func toNode(node api.Node) Node {
+func toNode(node api.Node, pods *api.PodList) Node {
+	allocatedResources, err := getNodeAllocatedResources(node, pods)
+	if err != nil {
+		log.Printf("Couldn't get allocated resources of %s node\n", node.Name)
```

Comment:
Same as above.

Will frontend hide additional columns on the list if allocatedResources are empty? They might be if there is an error which we only print and continue.